packages/components/src/SearchParamsSummary/SearchParamsSummary.js

```diff
@@ -41,40 +41,38 @@ export default function SearchParamsSummary({
 
   return (
     <View style={styles.container}>
-      <View style={styles.headerLeftcontainer}>
+      <View style={styles.headerLeftContainer}>
         <View style={styles.citiesContainer}>
           <Text style={styles.departureCity}>{departure?.city || ''}</Text>
           <Icon name={icon} />
           <Text style={styles.arrivalCity}>{arrival?.city || ''}</Text>
         </View>
-        {arrival != null && departure != null && (
-          <View>
-            {tripType === 'OneWay' ? (
-              <View>
-                <AdaptableBadge
-                  style={styles.badge}
-                  textStyle={styles.badgeText}
-                  text={arrival.localizedDate}
-                />
-              </View>
-            ) : (
-              <View style={styles.row}>
-                <AdaptableBadge
-                  style={styles.badge}
-                  textStyle={styles.badgeText}
-                  text={departure.localizedDate}
-                />
-                <Text style={styles.connector}> to </Text>
-                {/* @TODO localize string `to` */}
-                <AdaptableBadge
-                  style={styles.badge}
-                  textStyle={styles.badgeText}
-                  text={arrival.localizedDate}
-                />
-              </View>
-            )}
-          </View>
-        )}
+        {arrival != null &&
```

Comment:
There is a lot of logic here, and a ternary operator in JSX is not easy to parse. 
Would it make sense to create another component?

Reply:
I don't think it will be better with new component. What about something like this:
```
const isTripExists = arrival != null && departure != null;
...
{isTripExists && (tripType === 'OneWay' ? (...))}
```

Reply:
Well, `flow` is blind if I use this (^^^) notation with const.

Reply:
It should be now more clear: 
```
{arrival &&
          departure &&
          (tripType === 'OneWay' ? (
```